.github/workflows/main.yml

```diff
@@ -257,6 +257,11 @@ jobs:
           path: ~/.cache/coursier
           key: build-${{ inputs.spark }}-scala${{ inputs.scala }}-java${{ inputs.java }}-coursier
 
+      - name: Install R
```

Comment:
Perhaps this should be done in a preceding pr but let me try it out first.

Reply:
@dongjoon-hyun It seems that we need to manually install the R environment for ubuntu-latest, after the installation, the tests for 3.5.4 all seem to have succeeded.

Before

https://github.com/apache/spark-docker/actions/runs/12431196505/job/34721415046

After

https://github.com/apache/spark-docker/actions/runs/12436092928/job/34723236153?pr=77

Reply:
@dongjoon-hyun 

Is this solution acceptable? I referenced https://github.com/apache/spark/pull/48406. 

Should I submit a separate pr or can it be completed in this one?

also cc @Yikun @yaooqinn

Reply:
I make a separate one : https://github.com/apache/spark-docker/pull/78

Reply:
> I make a separate one : #78

Let me rebase this one